style.css

```diff
@@ -0,0 +1,93 @@
+.navbar {
+  height: 8vh;
+  top: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+  position: sticky;
+  background-color: #fff;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.hamburg {
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+
+.hamburg:hover {
+  transform: scale(1.2);
+  transition: all 0.7s;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 84vh;
+  background-image: url(assets/bg.svg);
+  padding: 20px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom-left-radius: 85px;
+  background-color: #fff;
+}
+
+.intor {
+  display: flex;
+  flex-direction: column;
+  height: 50%;
+  margin: 25px 0;
+  padding: 0 15px;
+}
+
+.title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  width: 75%;
+}
+
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 25px;
+  color: #344563;
+  padding: 15px 0 0 0;
+}
+
+.text {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+li {
+  list-style-type: none;
+}
+
+ul {
+  display: flex;
+  justify-content: space-between;
+  width: 50%;
+  padding-top: 15px;
+}
```

Comment:
- [ ] Kindly consider adding `align-items: center;` here to align these `icons` vertically, that would make it more organised and pretty looking.